workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+#if UNITY_EDITOR
+        private readonly LoggingDispatcher loggingDispatcher = new LoggingDispatcher();
+#endif
+
+        public void SetConnection(Connection connection)
```

Comment:
Set this in a constructor. This object shouldn't exist without one. The only case it could be null is if the worker disconnects, in which case we don't have a good flow for this.

Reply:
I disagree, we might create workers that don't connect immediately. Unless you want to tightly couple the creation of workers with the connection to Spatial.

Right now we create the dispatcher when we create the worker. This dispatcher could still log to Unity, even if no connection is set.